src/Npgsql/ConnectorPool.cs

```diff
@@ -599,15 +599,24 @@ void PruneIdleConnectors(object _)
         {
             var now = DateTime.UtcNow;
```

Comment:
@roji Just out of curiosity, have we ever considered using NodaTime internally?

Reply:
Any reason to use it?

Reply:
None in particular, just finished writing some code with NodaTime before looking at this, so it was fresh on my mind.

Reply:
Feel free to change it to what ever the Project standard is, for me it was only copy paste from: https://github.com/npgsql/npgsql/blob/a91fe0eb4674852b1ae079b81e1c90c3aadbf6e5/src/Npgsql/ConnectorPool.cs#L530

Reply:
The only reason not to use NodaTime internally is the added dependency - we want to force as little as possible upon users... Especially since there doesn't seem to bea lot of value in doing so here.

Reply:
Makes sense!